Source/WebCore/svg/SVGEllipseElement.cpp

```diff
@@ -81,6 +82,11 @@ void SVGEllipseElement::svgAttributeChanged(const QualifiedName& attrName)
 
 RenderPtr<RenderElement> SVGEllipseElement::createElementRenderer(RenderStyle&& style, const RenderTreePosition&)
 {
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+    if (document().settings().layerBasedSVGEngineEnabled())
+        return createRenderer<RenderSVGEllipse>(*this, WTFMove(style));
+#endif
```

Comment:
Duplicated from SVGCircleElement::createElementRenderer, but adding a common base class just for that seems overkill.

Reply:
Agreed.